code/game/objects/structures/tables_racks.dm

```diff
@@ -393,6 +399,7 @@
 			add_logs(user, src, "climbed onto")
 			user.Stun(2)

```

Comment:
You still need to unset  tableclimber here as well.

You need an unset both inside and outside of the if loop, thanks to the return 1 call.